src/main/java/org/folio/circulation/item/domain/entity/Item.java

```diff
@@ -0,0 +1,41 @@
+package org.folio.circulation.item.domain.entity;
+
+import java.util.UUID;
+
+
+import jakarta.persistence.Convert;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import org.folio.circulation.item.domain.converter.UUIDConverter;
+import org.folio.circulation.item.domain.entity.base.AuditableEntity;
+
+@Entity
+@Table(name = "circulation_item")
+@Getter
+@Setter
+@ToString
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class Item extends AuditableEntity {
+
+  @Id
+  @Convert(converter = UUIDConverter.class)
+  private UUID id;
+
+  private  UUID holdingsRecordId;
```

Comment:
I can see we are persisting holdingRecordsId, materialTypeId,permanentLoanTypeId at the time of circulation Item creation. I am assuming this api will be called by mod-dcb mostly. But, mod-dcb will not aware of the above values and those values will be null. In this case, how we are going to return a valid values for the above fields in the get API. I think we need to think on this aspect.

Reply:
perfect catch, @Vignesh-kalyanasundaram  !

But I don't think, this should be done in the scope of this work.
I believe it should be separated into a particular story or dev task.

@gurleenkaurbp , suggest pls